### PR TITLE
Introduce network-macvtap test configuration

### PR DESCRIPTION
**Pull request by iderzhav**
Signed-off-by: Igor Derzhavets igord@daynix.com

### CODE REVIEW

File: VirtHCK/hck_setup.cfg [hunk 1]
```diff
@@ -27,7 +27,8 @@ then
 fi
 
 #TEST DEVICE TYPE
-TEST_DEV_TYPE=network
+#TEST_DEV_TYPE=network
+TEST_DEV_TYPE=network-macvtap
```
Comment (YanVugenfirer) [inline VirtHCK/hck_setup.cfg]: Please don't make macvtap configuration default

**Comment by igord-daynix** [comment]
Please review update

**Comment by igord-daynix** [comment]
Please review final version

**Comment by dmitryfleytman** [comment]
Merged

**Pull request closed without merging**